package.json

```diff
@@ -16,10 +19,11 @@
     "clean": "gts clean",
     "codecov": "c8 report --reporter=json && codecov -f coverage/*.json",
     "fix": "gts fix",
-    "compile": "tsc -p .",
+    "compile": "tsc --target ES5 --module CommonJS",
+    "build-web": "microbundle",
```

Comment:
`microbundle` uses the same `tsconfig.json` file now, expect `outDir` is set by `package.json` **main**, **module** and **unpkg** fields